common_rest/src/main/java/uk/gov/dhsc/htbhf/ObjectMapperFactory.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.dhsc.htbhf;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class ObjectMapperFactory {
+
+    /**
+     * Configures an object mapper to use ISO local date and time formats and ignores unknown properties in json object.
+     * @return a configured object mapper
+     */
+    public static ObjectMapper configureObjectMapper() {
```

Comment:
This has been extracted from RequestContextConfiguration so we can use the same configuration for the ObjectMapper used by hibernate in the claimant service.

Reply:
I hate to duplicate a big chunk of code like this from a common module, can we refactor it so that we can reuse this code rather than duplicate please?